applications/sofa/gui/qt/CMakeLists.txt

```diff
@@ -167,18 +167,31 @@ if(Qt5Core_FOUND)
     set(MOC_FILES_GROUP "Generated")
     qt5_wrap_ui(FORM_FILES ${UI_FILES})
     set(FORM_FILES_GROUP "Generated")
-endif()
 
-if(WIN32)
-    sofa_copy_libraries_from_targets(${QT_TARGETS})
+    if(WIN32)
+        sofa_copy_libraries_from_targets(${QT_TARGETS})
+    endif()
     sofa_install_libraries_from_targets(${QT_TARGETS})
 
-    # Installing Qt platforms dll the hard way (dirty)
-    list (GET QT_TARGETS 0 QtCore_target)
-    get_target_property(QtCore_location ${QtCore_target} LOCATION_${CMAKE_BUILD_TYPE})
-    string(REGEX REPLACE "\\/[^/]*/[^/]*$" "" Qt_location ${QtCore_location})
-    FILE(GLOB QT_PLUGINS "${Qt_location}/plugins/platforms/qwindows*.dll")
-    install(FILES ${QT_PLUGINS} DESTINATION "bin/platforms" COMPONENT applications)
+    # QT INSTALL
```

Comment:
pour une install des libs qt un peu moins dirty: https://github.com/cgogn/SCHNApps/blob/develop/schnapps/CMakeLists.txt
ps: j'ai qu'une target windows, faudrait faire la même pour mac

Reply:
On my TODO list for a long long time...